pmm-app/tests/QAN/pages/qanPage.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
     table: '//table//tr[2]',
     tableRow: 'div.tr',
     noQueries: "//h1[contains(text(), 'No queries available for this combination of filters')]",
+    Last12HoursValue: "//span[contains(text(), 'Last 12 hours')]",
```

Comment:
This should already be available in DashboardPage, if not, thats the right place for such locator.